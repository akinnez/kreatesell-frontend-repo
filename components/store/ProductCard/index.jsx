import Image from 'next/image';
import {useRouter} from 'next/router';

import {Spin} from 'antd';

import styles from '../../../public/css/product-store.module.scss';
import {ExternalLink} from 'utils';

/**
 *
 * @param {*} CLink
 * @description - adds image transformations to the cloudinary link
 * @returns
 */
function formatCloudinaryImage(CLink) {
	if (CLink.includes('res.cloudinary.com')) {
		let linkarr = CLink.split('/');
		linkarr.splice(6, 0, 'q_auto:low');
		return linkarr.join('/');
	}
	return CLink;
}
const ProductCard = ({
	productDetails,
	sellingPrice,
	originalPrice,
	storename,
	convertedCurrency,
	targetCurrency,
	loading,
	handleModalOpen,
}) => {
	const pricingType =
		productDetails?.product_details?.pricing_type?.price_type;
	const router = useRouter();
	//=========================================================================//
	//==========================Images=========================================//
	//=========================================================================//

	// NOTE: get only the images
	const images = productDetails?.product_images?.filter(
		(img) =>
			img.file_type_name !== 'ContentZipFile' &&
			!['.rar', '.zip', '.pdf'].includes(img.filename)
	);

	// NOTE: for products with multiple images, it is a string seperated by a comma, so we are getting the first image
	const imageShown = images[0]?.filename?.split(',')[0];

	// there are instances where imageshown does not exist and image rendered is in a bad format (.i.e. starts with ,)
	// let len = imageRendered?.split(',');
	//=========================================================================//
	//==========================Images ENDS====================================//
	//=========================================================================//
	const statusLabel = {
		'In Stock': {color: '#2DC071'},
		'Out of Stock': {color: '#FF4D4F'},
	};

	/**
	 *
	 * @param {*} type - can be Pay What You Want or 'Fixed Price'
	 * @returns the predefined price for a particular currency
	 */
	const getPredefinedPrice = (type, priceType = 'Selling') => {
		let predefinedAmount;
		if (type === 'Pay What You Want') {
			predefinedAmount = productDetails?.check_out_details?.find(
				(det) =>
					det?.price_indicator === 'Minimum' &&
					det?.currency_name === targetCurrency
			);
			// minimum
		} else if (type === 'Fixed Price') {
			//selling
			predefinedAmount = productDetails?.check_out_details?.find(
				(det) =>
					det?.price_indicator === priceType &&
					det?.currency_name === targetCurrency
			);
		}
		return predefinedAmount?.price;
	};

	const outOfStock = () => {
		return (
			productDetails.number_sold >=
				productDetails.product_details?.number_of_product &&
			productDetails?.product_details?.is_limited_sales
		);
	};

	const showItemsLeftOrAmtSold = () => {
		let itemsLeft =
			productDetails?.number_of_product - productDetails?.number_sold;
		if (itemsLeft <= 10) {
			return `${itemsLeft} copies left!`;
		}
		if (productDetails.product_details?.is_show_number_of_sales) {
			return `${productDetails?.number_sold} sold`;
		}
	};
	if (!images[0]?.filename) return null;
	return (
		<div
			className={`bg-white w-full rounded-lg ${styles.productCardCtn}`}
			style={{cursor: 'pointer'}}
			onClick={() => {
				router.push(
					`/store/${storename}/product/${productDetails?.product_details?.kreasell_product_id}`
				);
			}}
		>
			<div className={styles.imageContainer}>
				<Image
					src={formatCloudinaryImage(imageShown)}
					width="320"
					height="300"
					className="rounded-t-lg object-cover"
					alt=""
					layout="responsive"
					quality={100}
				/>
			</div>
			<div className={`flex justify-between p-2 ${styles.productStatus}`}>
				<p
					className={`mb-0 ${styles.status}`}
					style={{
						color: statusLabel[
							outOfStock()
								? 'Out of Stock'
								: productDetails.status
						].color,
					}}
				>
					{/* if productDetails.total >= productDetails.number_sold : "Out of stock"*/}
					{outOfStock() ? 'Out of Stock' : productDetails.status}
				</p>
				<p className={`mb-0 ${styles.amountSold}`}>
					{' '}
					{showItemsLeftOrAmtSold()}
				</p>
			</div>
			<div className={`w-full px-2 md:px-4 ${styles.cardBox}`}>
				<p className={`pt-2 mb-1 text-sm md:text-base `}>
					{productDetails?.product_details?.product_name}
				</p>

				<div className={`flex justify-between items-center pb-4 pt-1`}>
					{productDetails?.product_price_type === 'Make it Free' ? (
						<p className={styles.makeItFreeText}>Free</p>
					) : productDetails?.product_price_type ===
					  'Pay What You Want' ? (
						<p
							className={`mb-0 text-base-gray ${styles.sellingPrice}`}
						>
							{targetCurrency ||
								productDetails?.default_currency?.currency}{' '}
							{getPredefinedPrice(
								productDetails?.product_price_type
							)
								? getPredefinedPrice(
										productDetails?.product_price_type
								  )
								: convertedCurrency
								? new Intl.NumberFormat().format(
										Number(
											convertedCurrency *
												productDetails.default_price
										).toFixed(2)
								  )
								: Number(productDetails.default_price).toFixed(
										2
								  )}
						</p>
					) : (
						<div
							className={`flex justify-between items-center column ${styles.main}`}
						>
							{loading ? (
								<Spin size="small" />
							) : (
								<>
									<p
										className={`text-base-gray text-sm md:text-base mb-0 ${styles.sellingPrice}`}
									>
										{targetCurrency ||
											productDetails?.default_currency
												?.currency}{' '}
										{getPredefinedPrice(
											productDetails?.product_price_type
										)
											? getPredefinedPrice(
													productDetails?.product_price_type
											  )
											: convertedCurrency
											? new Intl.NumberFormat().format(
													Number(
														convertedCurrency *
															sellingPrice
													).toFixed(2)
											  )
											: !convertedCurrency
											? new Intl.NumberFormat().format(
													Number(
														sellingPrice
													).toFixed(2)
											  )
											: '0.00'}
									</p>

									{pricingType === 'Fixed Price' &&
										originalPrice && (
											<p
												className={`text-base-gray  text-sm md:text-base originalPrice ${styles.originalPrice}`}
											>
												{targetCurrency ||
													productDetails
														?.default_currency
														?.currency}{' '}
												{getPredefinedPrice(
													productDetails?.product_price_type,
													'Original'
												)
													? getPredefinedPrice(
															productDetails?.product_price_type,
															'Original'
													  )
													: convertedCurrency
													? new Intl.NumberFormat().format(
															Number(
																convertedCurrency *
																	originalPrice
															).toFixed(2)
													  )
													: !convertedCurrency
													? new Intl.NumberFormat().format(
															Number(
																originalPrice
															).toFixed(2)
													  )
													: '0.00'}
											</p>
										)}
								</>
							)}
						</div>
					)}
					<Image
						alt=""
						src={ExternalLink}
						onClick={(e) => {
							e.stopPropagation();
							handleModalOpen(
								productDetails?.product_details
									?.kreasell_product_id
							);
						}}
					/>
				</div>
			</div>
		</div>
	);
};

export default ProductCard;
