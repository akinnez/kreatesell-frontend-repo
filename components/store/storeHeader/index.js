import React from "react";
import styles from "./Index.module.scss";
import {
	EditIcon,
	Facebook,
	ShareIcon,
	ViewAs,
	Twitter,
	Instagram,
	LinkedIn,
} from "../../IconPack";
import Dropdown from "../../dropdown";
import Router from "next/router";
import Social from "./social-media";
import { useSelector } from "react-redux";

const CtaButton = ({ Icon = () => <></>, label, active }) => {
	return (
		<>
			<div className={`cta ${active ? "active" : ""}`}>
				<Icon /> {label}
			</div>
			<style jsx>{`
				.cta {
					font-size: 14px;
					display: flex;
					align-items: center;
					gap: 5px;
					padding: 2px 10px;
					border-radius: 8px;
					cursor: pointer;
				}

				.cta.active {
					box-shadow: 0px 4px 6px -1px rgba(0, 0, 0, 0.12),
						0px 2px 4px -1px rgba(0, 0, 0, 0.07);
				}
			`}</style>
		</>
	);
};

export const ProtectedStoreHeader = ({
	storeName = "",
	coverImage = "",
	displayPicture = "",
	brandName = "",
	social = {},
	publicStore = false,
	publicStoreInfo = {},
}) => {
	return (
		<>
			<div
				className={styles.bg_wrapper}
				style={{
					backgroundImage: `url(${
						coverImage ||
						publicStoreInfo?.cover_page ||
						"/images/placeholder-1.jpg"
					})`,
				}}
			>
				<div className={styles.inner}>
					<div className={styles.inner_item_profile}>
						<div className={styles.profile_wrapper}>
							<div
								className={styles.image_intro_text}
								style={{
									backgroundImage: `url(${
										displayPicture ||
										publicStoreInfo?.display_picture ||
										"/images/placeholder-2.jpg"
									})`,
								}}
							/>
							<div className={styles.txt_wrapper}>
								<h3>
									{(brandName || publicStoreInfo?.brand_name) ?? "Olumide John"}
								</h3>
								<p>
									https://kreatesell.com/store/
									{storeName || publicStoreInfo?.store_name}
								</p>
							</div>
						</div>

						{!publicStore && (
							<div className={styles.cta_link_wrapper}>
								<ul>
									<li
										onClick={() => Router.push("/account/kreator/store/edit")}
									>
										<CtaButton Icon={EditIcon} label="Edit Profile" active />
									</li>
									<li>
										<Dropdown
											Button={<CtaButton Icon={ShareIcon} label="Share Link" />}
										>
											<Social
												facebook={social?.facebook}
												twitter={social?.twitter}
												instagram={social?.instagram}
												linkedIn={social?.linkedIn}
											/>
										</Dropdown>
									</li>
									<li>
										<CtaButton Icon={ViewAs} label="View As" />
									</li>
								</ul>
							</div>
						)}

						{publicStore && (
							<div className="hidden md:flex items-center px-10 lg:px-20">
								<div>Connect with me on</div>
								<div className="flex items-center justify-center lg:justify-between gap-4 pl-3">
									<a href={publicStoreInfo?.facebook ?? "#"}>
										<Facebook />
									</a>
									<a href={publicStoreInfo?.instagram ?? "#"}>
										<Instagram />
									</a>
									<a href={publicStoreInfo?.linked_ln ?? "#"}>
										<LinkedIn />
									</a>
									<a href={publicStoreInfo?.twitter ?? "#"}>
										<Twitter />
									</a>
								</div>
							</div>
						)}
					</div>
				</div>
			</div>

			{publicStore && (
				<div className="flex md:hidden items-center px-6 justify-center mt-16">
					<div>Connect with me on</div>
					<div className="flex items-center justify-center lg:justify-between gap-4 pl-3">
						<a href={publicStoreInfo?.facebook ?? "#"}>
							<Facebook />
						</a>
						<a href={publicStoreInfo?.instagram ?? "#"}>
							<Instagram />
						</a>
						<a href={publicStoreInfo?.linked_ln ?? "#"}>
							<LinkedIn />
						</a>
						<a href={publicStoreInfo?.twitter ?? "#"}>
							<Twitter />
						</a>
					</div>
				</div>
			)}
		</>
	);
};

export const StoreHeader = () => {
	return (
		<>
			<div
				className={styles.bg_wrapper}
				style={{ backgroundImage: `url(/images/placeholder-1.jpg)` }}
			>
				<div className={styles.inner}>
					<div className={styles.inner_item_profile}>
						<div className={styles.profile_wrapper}>
							<div
								className={styles.image_intro_text}
								style={{ backgroundImage: `url(/images/placeholder-2.jpg)` }}
							/>
							<div className={styles.txt_wrapper}>
								<h3>Olumide John</h3>
								<p>https://kreatesell.com/olumidejohn</p>
							</div>
						</div>
						<div className={styles.cta_link_wrapper}>
							<ul>
								<li onClick={() => Router.push("/account/store/edit")}>
									<CtaButton Icon={EditIcon} label="Edit Profile" active />
								</li>
								<li>
									<Dropdown
										Button={<CtaButton Icon={ShareIcon} label="Share Link" />}
									/>
								</li>
								<li>
									<CtaButton Icon={ViewAs} label="View As" />
								</li>
							</ul>
						</div>
					</div>
				</div>
			</div>
		</>
	);
};
