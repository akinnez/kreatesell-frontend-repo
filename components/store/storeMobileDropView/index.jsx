import Image from 'next/image';

import {Input} from 'antd';
import {Button} from 'components';

const StoreMobileDropView = ({
	isVariant = false,
	nameOfStore = '',
	dp = '',
}) => {
	const {singleStoreDetails} = useSelector((state) => state.product);

	const displayPicture = singleStoreDetails?.display_picture;

	const storeName = singleStoreDetails?.store_name;
	return (
		<div
			className={`${styles.mobileDropView} ${
				isVariant ? styles.isVariant : ''
			}`}
		>
			<div
				className={`${styles.profile} ${
					!displayPicture || !dp ? styles.noDp : ''
				}`}
			>
				{displayPicture || dp ? (
					<Image
						src={displayPicture || dp}
						alt="dp"
						layout="fill"
						className={styles.image}
					/>
				) : (
					<div className={styles.image_intro_text}>
						{/* <Avatar
							shape="square"
							className={styles.avatar}
							size={70}
							icon={<UserOutlined />}
						/> */}
					</div>
				)}
				<p>{storeName || nameOfStore}</p>
			</div>
			<div className={styles.storeLink}>
				<span>store link </span>
				<Image src={ExternalLink2} alt="external link" />
			</div>
			<div className={styles.text}>
				<h2>
					Host your <span>Digital Product</span> <br />
					online under minutes.
				</h2>
				<p>
					Seamlessly sell your content to audience without any
					marketing knowledge
				</p>
			</div>
			<div>
				<div className={styles.mobileInput}>
					<Input type="" placeholder="Enter your email.." />
				</div>
				<div
					className={styles.mobileButton}
					onClick={() => router.push('/signup')}
				>
					<Button
						text="Get Started Free"
						bgColor="blue"
						className={styles.freeBtn}
					/>
				</div>
				<div className={styles.benefits}>
					<span className={styles.benefitSpan}>Signup for free</span>
					<span className={styles.benefitSpan}>• Easy setup</span>
					<span className={styles.benefitSpan}>• Fast payout</span>
				</div>
			</div>
		</div>
	);
};

export default StoreMobileDropView;
