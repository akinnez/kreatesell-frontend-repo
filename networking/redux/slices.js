import { createSlice } from '@reduxjs/toolkit'

const initialState = {
data: [],
}

export const storeSlice = createSlice({
  name: 'kreatestore',
  initialState,
  reducers: {
    addItem:(state,action)=>{
        state = state.data.push(action.payload)
    }
  },
})

// Action creators are generated for each case reducer function
export const { addItem } = storeSlice.actions

export default storeSlice.reducer