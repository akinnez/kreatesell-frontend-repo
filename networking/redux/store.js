import { configureStore } from '@reduxjs/toolkit'
import StoreReducer from './slices'

export const store = configureStore({
  reducer: {
      store:StoreReducer
  },
})