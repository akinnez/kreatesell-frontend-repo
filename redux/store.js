import { configureStore } from '@reduxjs/toolkit'
import pageSlice from './slices/pageSlices'

const store = configureStore({
  reducer: {
      page: pageSlice
  },
})

export default store