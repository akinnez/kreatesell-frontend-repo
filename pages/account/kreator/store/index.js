import React,{useState,useEffect} from 'react'
import { Row, Column,Divider } from '../../../../components/grid'
import styles from '../../../../public/css/Store.module.scss'
import AuthLayout from "../../../../components/authlayout"
import {Button} from '../../../../components/inputPack'
import {Card} from '../../../../components/card'
import { CircularProgressbar,buildStyles  } from 'react-circular-progressbar';
import 'react-circular-progressbar/dist/styles.css'
import List from '../../../../components/list'
import Router from 'next/router'
import {ProtectedStoreHeader} from '../../../../components/store/storeHeader'
import useSWR  from 'swr'
import fetcher from '../../../../utils/fetcher'


const cardStyles = {
    borderRadius:"8px",
    padding:"35px 30px"
}

const progressbarStyles = buildStyles({
    rotation: 0.30,
   strokeLinecap: 'round',
   textSize: '24px',
   fontWeight:600,
   pathTransitionDuration: 0.5,
   pathColor: '#0072EF',
   textColor: '#595959',
   trailColor: '#E6F7FF'
 })



const Index = ()=>{

 
  
    const [loading, setLoading] = useState(false)

    const {data} = useSWR('v1/kreatesell/store/me',fetcher )

    // useEffect(()=>{
    //     setLoading(true)
    //     ApiService.request(
    //         'get',
    //         'v1/kreatesell/store/me',
    //     ({data})=>{
    //         setData(data)
    //         setLoading(false)
    //     })
    // },[])
  

    return(
        <>
        <AuthLayout loading={loading}>
           
          <ProtectedStoreHeader 
            brandName={data?.store_details?.brand_name}
            storeName={data?.store_details?.store_name}
            coverImage={data?.store_details?.cover_page}
            displayPicture={data?.store_details?.display_picture}
            social={{
                facebook:data?.store_details?.facebook,
                twitter:data?.store_details?.twitter,
                instagram:data?.store_details?.instagram,
                linkedIn:data?.store_details?.linked_ln,
                }}/>
            <Row style={{marginTop:"100px"}}>
                <Column m="7" s="12">
                    <Card style={cardStyles}>
                        <div className={styles.bio_info}>
                            <h5>Description</h5>
                            <p>{data?.store_details?.bio_data}</p>
                         </div>
                    </Card>
                </Column>
                <Column m="5">
                <Card  style={{...cardStyles}}>
                <div className={styles.progress_wrapper}>
                    <div className={styles.progress}>
                        <CircularProgressbar 
                            text={data?.percentage_completed+'%'}
                            value={data?.percentage_completed}
                            strokeWidth={15}
                            styles={progressbarStyles} />
                    </div>
                    <div id={styles.progress_text}>
                        <p>You've completed <strong>{data?.percentage_completed}%</strong> of your store setup</p>
                    </div>
                </div>
                <Divider />

                <List step={data?.percentage_completed <= 40 ? 0: data?.percentage_completed > 40 && data?.percentage_completed < 100 ? 1:3} 
                    list={[
                    "Complete your store profile details",
                    "Add your bank account details to receive your payments",
                    "Add your first product to increase your store completion"]}
                    />
                </Card>
                </Column>
            </Row>

            <Row>
                <Column m="12" s="12" align="center" style={{marginTop:"20px"}}>
                    <p>Almost there, now click the button to add your product</p>
                        <Button label="+ Add Product" style={{marginTop:"20px"}} onClick={()=>Router.push("#")}/>
                </Column>
            </Row>
         
        
        </AuthLayout>
       
        
        </>
    )
}

export default Index