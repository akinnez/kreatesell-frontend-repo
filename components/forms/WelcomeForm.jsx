/* eslint-disable react/no-unescaped-entities */
import { useState } from "react";
import { useFormik } from "formik";
import { Button, Input, FormError } from "../";
import { StoreNameSchema, BusinessNameSchema } from "../../validation";
import styles from "../../public/css/Welcome.module.scss";
import { isAnEmpytyObject } from "../../utils";
import { WelcomeStoreOnboarding } from "../../redux/actions";
import { useSelector } from "react-redux";
import { useRouter } from "next/router";

export const WelcomeForm = () => {
	const router = useRouter();
	const welcomeStoreOnboarding = WelcomeStoreOnboarding();
	const [step, setStep] = useState(1);

	const { loading } = useSelector((state) => state.store);

	const initialValues = {
		Brand_Name: "",
		Store_Name: "",
	};

	const handleSubmit = async (data) => {
		if (step === 1) {
			setStep(2);
		} else if (step === 2) {
			await welcomeStoreOnboarding(data, () => {
				router.push("/account/dashboard");
			});
		}
	};

	const formik = useFormik({
		initialValues,
		onSubmit: handleSubmit,
		validationSchema: step === 1 ? BusinessNameSchema : StoreNameSchema,
		validateOnChange: false,
	});

	return (
		<form
			className={styles.welcome}
			onSubmit={formik.handleSubmit}
			autoComplete="off"
		>
			<h5 className={styles.mobileTitle}>
				{step === 1 && `Tell us a little about <br /> yourself`}
				{step === 2 && `Here is what your store link looks like`}
			</h5>
			<h5 className={styles.webTitle}>
				{step === 1 && `Tell us a little about yourself`}
				{step === 2 && `Here is what your store link looks like`}
			</h5>
			<p className={styles.subTitle}>
				{step === 1 &&
					`Now that you're all signed up, let's personalize your store.`}
				{step === 2 && `Take time to verify `}
			</p>

			{!isAnEmpytyObject(formik.errors) && <FormError errors={formik.errors} />}

			<p className={styles.label}>
				{step === 1 &&
					`1. Your Business Name - will be displayed on your store`}
				{step === 2 && `2. Personalize your store link.`}
			</p>

			{step === 1 && (
				<Input
					placeholder="Enter name here ... "
					name="Brand_Name"
					className={styles.input}
					onChange={formik.handleChange}
				/>
			)}

			{step === 2 && (
				<div className="mb-12">
					<Input
						placeholder="Enter name here ... "
						name="Store_Name"
						className={styles.input}
						onChange={formik.handleChange}
					/>

					{formik.values.Store_Name && (
						<span className={styles.storePreviewLink}>
							{`https://kreatesell.com/${formik.values.Store_Name}`}
						</span>
					)}
				</div>
			)}

			<Button
				type={step === 2 ? "submit" : "click"}
				text={step === 2 ? "Start Selling" : "Continue"}
				bgColor="primaryBlue"
				className={styles.button}
				loading={loading}
			/>
		</form>
	);
};
