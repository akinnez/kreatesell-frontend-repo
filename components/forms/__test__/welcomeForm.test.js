import { WelcomeForm } from "../WelcomeForm";
import renderer from "react-test-renderer";
import * as nextRouter from "next/router";
import { Provider } from "react-redux";
import { initializeStore } from "../../../redux/store";

nextRouter.useRouter = jest.fn();
nextRouter.useRouter.mockImplementation(() => ({
  route: "/",
}));
const mockState = { auth: "", domain: "", store: "", utils: "", product: "" };

it("renders a snapshot of <WelcomeForm/>", () => {
  const form = renderer
    .create(
      <Provider store={initializeStore(mockState)}>
        <WelcomeForm />
      </Provider>
    )
    .toJSON();
  expect(form).toMatchSnapshot();
});
