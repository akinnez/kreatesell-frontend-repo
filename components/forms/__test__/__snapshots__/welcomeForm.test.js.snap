// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot of <WelcomeForm/> 1`] = `
<form
  autoComplete="off"
  className="welcome"
  onSubmit={[Function]}
>
  <h5
    className="mobileTitle"
  >
    Tell us a little about &lt;br /&gt; yourself
  </h5>
  <h5
    className="webTitle"
  >
    Tell us a little about yourself
  </h5>
  <p
    className="subTitle"
  >
    Now that you're all signed up, let's personalize your store.
  </p>
  <p
    className="label"
  >
    1. Your Business Name - will be displayed on your store
  </p>
  <div
    className="undefined inputContainer"
  >
    <label
      className="label undefined"
      htmlFor="Brand_Name"
    />
    <input
      className="input false false input"
      name="Brand_Name"
      onChange={[Function]}
      placeholder="Enter name here ... "
    />
  </div>
  <button
    className="
			button
			button
			false 
			false
			primaryBlue
			"
    type="click"
  >
    <div
      className="buttonValue"
    >
      Continue
    </div>
  </button>
</form>
`;
